addons/sale_timesheet/views/sale_order_views.xml

```diff
@@ -9,11 +9,11 @@
                     <xpath expr="//button[@name='action_view_invoice']" position="before">
                         <field name="timesheet_count" invisible="1" />
                         <button type="object"
-                           name="action_view_timesheet"
-                           class="oe_stat_button"
-                           icon="fa-clock-o"
-                           attrs="{'invisible': [('timesheet_count', '=', 0)]}"
-                           groups="hr_timesheet.group_hr_timesheet_user">
+                            name="action_view_timesheet"
+                            class="oe_stat_button"
+                            icon="fa-clock-o"
+                            attrs="{'invisible': [('timesheet_count', '=', 0)]}"
+                            groups="hr_timesheet.group_hr_timesheet_user">
```

Comment:
Same remark, why do you reformat these lines ? 🤔

Reply:
I believe this was VScode that automatically did this for some reason. I will try to investigate why this happened :)